app/move/validators.js

```diff
@@ -1,6 +1,26 @@
 const { parseISO, isValid, isAfter } = require('date-fns')
 const { Controller } = require('hmpo-form-wizard')
 
+function isPNCNumberValid(value) {
+  value = (value || '').toString().toUpperCase()
+  const regex = /^([0-9]{2}|[0-9]{4})\/[0-9]+[a-zA-Z]$/i
+  const mod23chars = 'Z-A-B-C-D-E-F-G-H-J-K-L-M-N-P-Q-R-T-U-V-W-X-Y'.split('-')
+  let validPNC = false
+
+  if (Controller.validators.regex(value, regex)) {
+    /* Use CJSE Data Standards Catalogue v4.3, section 3.374 */
+    const derivedPNC = value.replace('/', '').substring(2, 11)
+    const checkDigit = value.substring(12)
+    const i = Number.parseInt(derivedPNC) % 23
+
+    if (mod23chars[i] === checkDigit) {
+      validPNC = true
```

Comment:
I know this is copied from Confluence, but I think it'd be nicer to do away with validPNC and just return true here